resources/repos.json

```
@@ -1005,6 +1005,22 @@
             }
         }
     },
+    "waleks/SMG:master": {
+        "sonar": {
+            "enabled": false
+        },
+        "options": {
+            "prefix": "DEV"
+        },
+        "dependencies": {
+            "Minecraft Version(s)": {
+                "1": "1.16.x"
+            },
+            "Slimefun Version": {
+                "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#21\">RC-21</a>"
+            },
```

Also this comma shouldn't be here.
```suggestion
            }
```